simtools/applications/db_development_tools/add_new_parameter_from_json_file_to_db.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application is used to add a new parameter to the sites collection in the DB
+    using a json file as input.
+
+    This application should not be used by anyone but expert users and not often.
+    Therefore, no additional documentation about this applications will be given.
+
+"""
+
+import logging
+
+import simtools.utils.general as gen
+from simtools.configuration import configurator
+from simtools.db import db_handler
+
+
+def main():
+    config = configurator.Configurator(description=("Add a new parameter to the DB."))
+    config.parser.add_argument("--file_name", help="file to be added", required=True)
+    config.parser.add_argument(
+        "--db_collection", help="DB collection file will be added", required=True
+    )
+    args_dict, db_config = config.initialize(db_config=True)
+
+    logger = logging.getLogger()
+    logger.setLevel(gen.get_log_level_from_user(args_dict["log_level"]))
+
+    db = db_handler.DatabaseHandler(mongo_db_config=db_config)
+
+    if args_dict["db_collection"] not in ("telescopes", "sites"):
```

Comment:
Why are you limiting to those specific ones? Because they are only ones which the specific schema you are using? 
In general I am not crazy about hardcoding these names here. Maybe you can get the available names from `db`? Or perhaps at that check to the `add_new_parameter` function? 
Ahh it's actually already in that function, so this test can be removed from here.

Reply:
Fixed.